applications/DAOsign.md

```diff
@@ -0,0 +1,183 @@
+
+**Misha Kushka**: Tech Lead and Blockchain Developer
+Background: 6+ years of professional experience as a developer, 4+ years of professional experience in the Blockchain field, 4+ years experience as a lead
+
+### Team Code Repos
+
+- https://github.com/ConsiderItDone
+- https://github.com/DAOsign
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- [Eugene Fine](https://github.com/ConsiderItDone)
+- [Ramil Amerzyanov](https://github.com/ramilexe)
+- [Oleksandra Burmenska](https://github.com/Burmenska)
+- [Misha Kushka](https://github.com/kushkamisha)
+
+### Team LinkedIn Profiles (if available)
+
+- [Eugene Fine](https://www.linkedin.com/in/eugenefine/)
+- [Ramil Amerzyanov](https://www.linkedin.com/in/ramil-amerzyanov/)
+- [Oleksandra Burmenska](https://www.linkedin.com/in/oleksandra-burmenska/)
+- [Misha Kushka](https://www.linkedin.com/in/mkushka/) 
+
+
+## Development Status :open_book:
+
+We are currently working on a tech demo (kind of a pre-beta version) and non-blockchain version of DAOsign is available here: https://testnet.daosign.org
+
+## Development Roadmap :nut_and_bolt:
+
+**Overview**
+
+Total Estimated Duration: 2 months
+Full-Time Equivalent (FTE): 2 FTE per month
+Total Costs: 48,100 USD
+
+Milestone 1 - Smartcontracts. 
+Total Costs: 40,300 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0 |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can upload/generate the agreement, add signers/observers and complete the signature process with storing signature proofs either in test or mainnet, which will show how the new functionality works. |
+| 0c. | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Delivery format | We will provide a website that will include the agreement creation, modification and signing functionality for users with further proofs storing on Polkadot chain.
+| 0e. | Article | We will publish an **article** that explains how does DAOsign users can create agreements and store them in Polkadot test- or mainnet. It will be published in our Medium, Linkedin, Twitter and Discord channels. Apart of that, we will regularly publish grant development status on our social media. 
+| 1. | Smart Contracts | Agreement Contract |
+| 2. | Smart Contracts | EIP712 standard implementation on Ink! reusing https://github.com/Brushfam/openbrush-contracts|
+| 3. | Smart Contracts | EIP2771 standard implementation on Ink! |
+| 4. | Smart Contracts | Delegate call using EIP2771 |
```

Comment:
Could you add the gas relayer to the list of deliverables? It'd be great to have a complete meta transaction setup to test and run.

Reply:
Added gas relayer component to the deliverables list